scripts/download_incr_DB.py

```diff
@@ -114,6 +114,13 @@ def GetStateDeltaFromS3(bucketName):
 	GetAllObjectsFromS3(getURL(bucketName), STATEDELTA_DIFF_NAME)
 	ExtractAllGzippedObjects()
 
+def IsDownloadRestartRequired(currTxBlk, latestTxBlk) :
+    print("currTxBlk = "+ str(currTxBlk) + " latestTxBlk = "+ str(latestTxBlk) + " NUM_DSBLOCK = " +str(NUM_DSBLOCK))
+    if((latestTxBlk // (NUM_DSBLOCK * NUM_FINAL_BLOCK_PER_POW)) != (currTxBlk // (NUM_DSBLOCK * NUM_FINAL_BLOCK_PER_POW))):
```

Comment:
Minor comment: Can you add some comments here on what this check means?

Reply:
adding.